site/content/overview/gateway-api-compatibility.md

```diff
@@ -10,18 +10,20 @@ Learn which Gateway API resources NGINX Gateway Fabric supports and to which lev
-| [GatewayClass](#gatewayclass)         | Supported          | Not supported          | Supported                             | v1          |
-| [Gateway](#gateway)                   | Supported          | Partially supported    | Not supported                         | v1          |
-| [HTTPRoute](#httproute)               | Supported          | Partially supported    | Not supported                         | v1          |
-| [ReferenceGrant](#referencegrant)     | Supported          | N/A                    | Not supported                         | v1beta1     |
-| [GRPCRoute](#grpcroute)               | Supported          | Partially supported    | Not supported                         | v1          |
-| [TLSRoute](#tlsroute)                 | Supported          | Not supported          | Not supported                         | v1alpha2    |
-| [TCPRoute](#tcproute)                 | Not supported      | Not supported          | Not supported                         | N/A         |
-| [UDPRoute](#udproute)                 | Not supported      | Not supported          | Not supported                         | N/A         |
-| [BackendTLSPolicy](#backendtlspolicy) | Supported          | Supported              | Not supported                         | v1alpha3    |
-| [Custom policies](#custom-policies)   | N/A                | N/A                    | Supported                             | N/A         |
+
+| Resource                              | Core Support Level | Extended Support Level | Implementation-Specific Support Level | API Version | API Release Channel |
+|---------------------------------------|--------------------|------------------------|---------------------------------------|-------------|---------------------|
+| [GatewayClass](#gatewayclass)         | Supported          | Not supported          | Supported                             | v1          | Standard            |
+| [Gateway](#gateway)                   | Supported          | Partially supported    | Not supported                         | v1          | Standard            |
+| [HTTPRoute](#httproute)               | Supported          | Partially supported    | Not supported                         | v1          | Standard            |
+| [GRPCRoute](#grpcroute)               | Supported          | Partially supported    | Not supported                         | v1          | Standard            |
+| [ReferenceGrant](#referencegrant)     | Supported          | N/A                    | Not supported                         | v1beta1     | Standard            |
+| [TLSRoute](#tlsroute)                 | Supported          | Not supported          | Not supported                         | v1alpha2    | Experimental        |
+| [TCPRoute](#tcproute)                 | Not supported      | Not supported          | Not supported                         | N/A         | N/A                 |
```

Comment:
Yea I noticed that too and that was a bit confusing, I understood it as, if the resource graduates / moves versions or channels -- since we haven't implemented it -- we might not notice it and thus would forget to update it here or something like that. 

Is that not of concern?

Reply:
That could be the reason. I don't think it's a huge concern though, since we are pinned to a specific version of the Gateway API. When we update to a new version, we just need to make sure this doc is updated.

Reply:
Should be added, can you double check that I have the api version and release channel correct?

Reply:
LGTM